LICENSE

```diff
@@ -67,6 +67,9 @@ The following 3rd party binaries are shipped with DOSBox Staging:
 - FreeDOS keyboard layout files in resources/freedos-keyboard are GPL-2.0 licensed;
   source code: https://gitlab.com/FreeDOS/base/keyb_lay
 
+The DOSBox Staging source code contains full snapshots of the original
```

Comment:
Technically, it's not our source code that includes them but our repository. Could you change that?